src/Microsoft.IdentityModel.Protocols.WsFederation/WsFederationMessage.cs

```diff
@@ -100,7 +101,7 @@ public WsFederationMessage(WsFederationMessage wsFederationMessage)
         {
             if (wsFederationMessage == null)
             {
-                Logger.WriteWarning(FormatInvariant(LogMessages.IDX22000, "wsfederationMessage"));
```

Comment:
Search for 'Logger.Write...' then

Reply:
Done! All of those should be replaced now.